app/views/forum/topic/form.scala.html

```diff
@@ -1,5 +1,8 @@
 @(categ: lila.forum.Categ, form: Form[_], captcha: lila.common.Captcha)(implicit ctx: Context)
 
+@jQueryTag
```

Comment:
Shouldn't matter once I've moved it out to a submodule with jQuery as a dependency anyway!